ui/lib/kv/addon/components/page/secret/details.js

```diff
@@ -32,6 +33,13 @@ export default class KvSecretDetails extends Component {
     this.showJsonView = !this.showJsonView;
   }
 
+  @action
+  onClose(dropdown) {
+    // strange issue where closing dropdown triggers full transition (which redirects to auth screen in production)
```

Comment:
copied this comment from the original dropdown component used here https://github.com/hashicorp/vault/blob/e55c18ed1299e0d36b88e603fa9f12adaf8e75dc/ui/app/components/secret-version-menu.js#L18